simbad/rotsearch/phaser_search.py

```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
         summarize_result(self.search_results,
                          csv_file=csv_file, columns=columns)
 
+    @staticmethod
+    def calculate_ermsd(nres, id):
+        """Calculate the estimated rmsd based on the number of residues and sequence id"""
+
+        if id < 0 or id > 100:
```

Comment:
Non-essential but could re-write this to `0 > id < 100`. I think this works and is generally easier to read plus quicker if done many times (which it is not here but just in general)